CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.10.4
+
+- Add the `--api-url` option
+  This can be used to set the endpoint for the ngrok API.
+  It can be set through via the helm `apiURL` value.
```

Comment:
you think we can include https://github.com/ngrok/kubernetes-ingress-controller/pull/360 here?

Reply:
Ehhh... maybe let's break out the version bump into its own change and do both there. Didn't realize I would be pulling in other stuff.

Reply:
#363